py3status/modules/do_not_disturb.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 """
-A simple "Do Not Disturb" module that can turn on and off all system notifications.
+Do not disturb by turning on and off all system notifications.
```

Comment:
Should we keep "Do Not Disturb" to help people recognize it for what it is? From Apple. https://support.apple.com/en-us/HT204321